HACKING.md

```diff
@@ -0,0 +1,25 @@
+# Hacking tips for Akira
+
+## Formatting
+
+To avoid mess with codestyle we use uncrustify to parse and format files
+
+- How to check if all files are correctly formated:
+
+        find -name *.vala -exec uncrustify -c uncrustify.cfg \{} --check \; |& grep FAIL
+
+- How to format files before submit a MR:
```

Comment:
What is `MR`? 

Sorry, I have never seen it before. I only know `PR` i.e Pull Requests.

Reply:
Merge request. Is the term for Pull request in gitlab. Have to fix it

Reply:
TIL 😄